test/lib/rules/yield-effects-spec.js

```diff
@@ -24,10 +96,60 @@ ruleTester.run("yield-effects", rule, {
     }
   ],
   invalid: [
+    // overrides
+    {
+      code: buildTest("import { take } from 'redux-saga'", "const take = () => {}; yield take()"),
+      errors: [ERRORS.overrides]
+    },
+    {
+      code: buildTest("import { take } from 'redux-saga'; const take = () => {}", "yield take()"),
+      errors: [ERRORS.overrides]
+    },
+    {
+      code: buildTest("import { take } from 'redux-saga'; var take = function() {}", "yield take()"),
+      errors: [ERRORS.overrides]
+    },
+    {
+      code: buildTest("import { take } from 'redux-saga'; const take = function() {}", "yield take()"),
+      errors: [ERRORS.overrides]
+    },
+    {
+      code: buildTest("import { take } from 'redux-saga'; function take() {}", "yield take()"),
+      errors: [ERRORS.overrides]
+    },
+
+    // overrides and no yield at the same time
+    {
+      code: buildTest("import { take } from 'redux-saga'", "const take = () => {}; take()"),
+      errors: ERRORS.overrides_and_noyield
```

Comment:
I would consider that still an override error since the `take` override hides the saga effect import. That its most likely a serious error. If not, the user can still disable the rule for that section of the file.